### PR TITLE
PAYARA-3387 catalog translation also works for WEB-INF/jax-ws-catalog/xml

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/WebServicesDeployer.java [hunk 1]
```diff
@@ -182,8 +181,14 @@ protected void cleanArtifacts(DeploymentContext deploymentContext) throws Deploy
     }
 
     protected void setupJaxWSServiceForDeployment(DeploymentContext dc, WebService ws) throws DeploymentException {
-
-        BundleDescriptor bundle = dc.getModuleMetaData(BundleDescriptor.class);
+        
+        Collection<Object> allMetaData = dc.getModuleMetadata();
+        List<BundleDescriptor> bundleDescriptors = new ArrayList<>(2);
+        for (Object metaData :  allMetaData.toArray()) {
```
Comment (svendiedrichsen) [inline appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/WebServicesDeployer.java]: Why toArray()? Collections are iterable.

**Comment by Cousjava** [comment]
jenkins test please

**Comment by smillidge** [comment]
Why is this DO NOT MERGE?

**Comment by Cousjava** [comment]
tested with customer app and failed

**Pull request closed without merging**